packages/web-components/src/scripts/build-prod.js

```diff
@@ -1,50 +1,107 @@
 #!/usr/bin/env node
+const ejs = require('ejs')
 const path = require('path')
-const { runCommand } = require('../../../../scripts/release/utils')
-
-const releaseDevThemes = () => {
-  try {
-    const bucketName = 'reapit-web-components'
-    const themesDistPath = path.resolve(__dirname, '../../', 'public', 'themes')
-    // Copy new version to the bucket
-    runCommand('aws', [
-      's3',
-      'cp',
-      themesDistPath,
-      `s3://${bucketName}`,
-      '--grants',
-      'read=uri=http://acs.amazonaws.com/groups/global/AllUsers',
-      '--recursive',
-    ])
-  } catch (err) {
-    console.error(err)
-    throw new Error(err)
-  }
+const { writeFileSync } = require('fs')
+const { execSync, spawn } = require('child_process')
+const { setEnv } = require('./get-env')
+const formatCode = require('./format-code')
+
+// since the build process is executed in a parallel manner, folder of generated index files may not have been created
+// have to create it first using mkdirp (mkdir with cursive)
+const mkdirp = require('mkdirp')
+
+const cjsIndexFolderPath = path.resolve(__dirname, '../../public/dist-npm/cjs')
+const cjsIndexFilePath = path.resolve(cjsIndexFolderPath, './index.js')
+const cjsIndexTemplateFilePath = path.resolve(__dirname, './tpls/index.cjs.ejs')
+
+const esmIndexFolderPath = path.resolve(__dirname, '../../public/dist-npm/esm')
+const esmIndexFilePath = path.resolve(esmIndexFolderPath, './index.js')
+const esmIndexTemplateFilePath = path.resolve(__dirname, './tpls/index.esm.ejs')
+
+const tsDeclarationIndexFolderPath = path.resolve(__dirname, '../../public/dist-npm/types')
+const tsDeclarationIndexFilePath = path.resolve(tsDeclarationIndexFolderPath, './index.d.ts')
+const tsDeclarationIndexTemplateFilePath = path.resolve(__dirname, './tpls/index.d.ts.ejs')
+
+const generateFileForPackages = async ({ packages, tplPath, destinationFilePath, destinationFolderPath }) => {
+  const compileResult = await ejs.renderFile(tplPath, {
+    packages,
+  })
+
+  const formatResult = formatCode(compileResult)
```

Comment:
why we need to format code in production build, I thought we should minify it instead?